fe/fe-core/src/main/java/com/starrocks/catalog/ResourceGroup.java

```diff
@@ -127,6 +131,11 @@ private List<String> showClassifier(ResourceGroupClassifier classifier) {
             row.add("" + 0);
         }
         row.add("" + concurrencyLimit);
+        if (spillMemLimitThreshold != null) {
+            row.add("" + (spillMemLimitThreshold * 100) + "%");
```

Comment:
check it is valid?

Reply:
> check it is valid?

I think it's not necessary, because we have checked it when create/alter resource group